project/profiles/capacity4more/modules/c4m/og/c4m_og_invite_visitors/c4m_og_invite_visitors.module

```diff
@@ -0,0 +1,671 @@
+<?php
+
+/**
+ * @file
+ * Invite new members to a group or project.
+ */
+
+/**
+ * Implements hook_menu().
+ *
+ * Defines a path for the page/form used to invite visitors into a group.
+ */
+function c4m_og_invite_visitors_menu() {
+  $items = array();
+  $items['group/%/%/admin/people/invite-visitors'] = array(
```

Comment:
We need to keep the current path for the link to appear in the same sub-menu as 'Invite users' and `Add members`.